File: BackEnd/CanchaLibre/CanchaLibreRS/src/main/java/pe/edu/pucp/canchalibre/rs/dto/IngresoCanchaDTO.java
package pe.edu.pucp.canchalibre.rs.dto;

public class IngresoCanchaDTO {
    private int id;
    private String nombre;
    private double ingresos;
    private int reservas;
    private int horasOcupacion;

    public int getId() { return id; }
    public void setId(int id) { this.id = id; }
    public String getNombre() { return nombre; }
    public void setNombre(String nombre) { this.nombre = nombre; }
    public double getIngresos() { return ingresos; }
    public void setIngresos(double ingresos) { this.ingresos = ingresos; }
    public int getReservas() { return reservas; }
    public void setReservas(int reservas) { this.reservas = reservas; }
    public int getHorasOcupacion() { return horasOcupacion; }
    public void setHorasOcupacion(int horasOcupacion) { this.horasOcupacion = horasOcupacion; }
}

File: FrontEnd/CanchaLibre/CanchaLibreWeb/CanchaLibreWeb/Servicios/Admin/IngresoCanchaDto.cs
namespace CanchaLibreWeb.Servicios.Admin;

public sealed class IngresoCanchaDto
{
    public int Id { get; set; }
    public string Nombre { get; set; } = string.Empty;
    public double Ingresos { get; set; }
    public int Reservas { get; set; }
    public int HorasOcupacion { get; set; }
}
